types/types.go

```diff
@@ -738,15 +741,11 @@ func GetShareManagerNameFromShareManagerPodName(podName string) string {
 }
 
 func GetBackupVolumeNameFromVolumeName(volumeName string) string {
-	return generateBackupResourceName(volumeName)
-}
-
-func generateBackupResourceName(resourceName string) string {
-	return util.GetStringChecksum(strings.TrimSpace(resourceName))[:util.RandomIDLength] + "-" + util.RandomID()
+	return volumeName + backupVolumeSuffix + "-" + util.RandomID()
```

Comment:
`backupVolumeSuffix` seems not really required?

Reply:
Removed. Thanks